### PR TITLE
Removes cyborg as a joinable job

### PR DESCRIPTION
**Pull request by senorsapo**
Removes cyborg as a joinable job from the menu. It no longer appears in preferences, or the latejoin menu. I've playtested it with the menu + spawning a borg and playing around in it and it looked fine to me.

Please tell me if there's anything missing or gamebreaking. I've ran it and there were no errors, but there might be something I haven't caught and that might do a random runtime in specific circumstances.

https://forums.aurorastation.org/topic/17615-remove-cyborg/

edit forgot to add thread

Edit2 - I've edited the Pull Request so that cyborgs cannot be created. Once you reach the final step where you click on the completed frame with a hand, it tells you the connections are incompatible. IPCs can still be created, though.

### CODE REVIEW

**Comment by senorsapo** [comment]
!review

**Comment by DreamySkrell** [comment]
Should also remove the posibrains on the mechanist workshop. Otherwise effectively there are two borg slots in game still, just need a machinist to build them.

**Comment by Yakunan** [comment]
I strongly disagree. I have a favourite cyborg which i want to keep playing. Others might have too.

**Comment by alsoandanswer** [comment]
There should probably be input from the synthetic loredev on this

**Comment by niennab** [comment]
A change as huge as this should really be utilizing the server's vote function.

**Comment by FlamingLily** [comment]
> I've edited the Pull Request so that cyborgs cannot be created. Once you reach the final step where you click on the completed frame with a hand, it tells you the connections are incompatible. IPCs can still be created, though.

If this PR intends to disable the cyborgification procedure as a punishment for crimes generally, then this is code-wise fine (though definitely needs way more staff/CCIA/lore input)

If not, then this change has effectively rendered cyborgification impossible against IPCs and only IPCs, which is not good, as it's a blatant inconsistency (and also, still need way more staff/CCIA/lore input)

I'm sure it's very clear via the do not merge tag that this still needs staff input but I also think this change making it impossible to create cyborgs isn't thought out enough

**Comment by senorsapo** [comment]
@FlamingLily 
This makes it unable to slot in a posi or mmi into the completed cyborg body.

**Comment by Ben10083** [comment]
> @FlamingLily This makes it unable to slot in a posi or mmi into the completed cyborg body.

that isnt a soft removal then, that makes cyborgs impossible to do. Please update your PR title.

**Comment by SleepyGemmy** [comment]
> that isnt a soft removal then, that makes cyborgs impossible to do. Please update your PR title.

it makes cyborgs unplayable, it doesn't remove them from the game. hence, "soft" removal.

File: code/__defines/jobs.dm [hunk 1]
```diff
@@ -16,7 +16,7 @@
 #define CIVILIAN_ROLES list(/datum/job/assistant, /datum/job/visitor, /datum/job/passenger)
 #define SECURITY_ROLES list(/datum/job/warden, /datum/job/investigator, /datum/job/officer, /datum/job/intern_sec)
 #define OPERATIONS_ROLES list(/datum/job/hangar_tech, /datum/job/mining, /datum/job/machinist)
-#define EQUIPMENT_ROLES list(/datum/job/ai, /datum/job/cyborg)
```
Comment (SleepyGemmy) [inline code/__defines/jobs.dm:19]: [1]

**Comment by DreamySkrell** [comment]
>I've edited the Pull Request so that cyborgs cannot be created. Once you reach the final step where you click on the completed frame with a hand, it tells you the connections are incompatible. IPCs can still be created, though.

Agreed with Gem above. Cyborgs should still be buildable - by third parties, or with random posibrains found in the warehouse or away sites or the like. Only an easy way for players to consistently join as cyborgs should be removed.

**Comment by Arrow768** [comment]
Has been polled.
Closed as result of poll outcome/maintainer discussion.

**Pull request closed without merging**